words/disasm.asm

```diff
@@ -105,22 +104,16 @@ _byte_loop:
                 bpl _no_operand         ; bit 7 clear, single-byte instruction
 
                 ; We have an operand. Prepare the Data Stack
-                jsr w_zero             ; ( addr u 0 ) ZERO does not use Y
 
                 ; Because of the glory of a little endian CPU, we can start
                 ; with the next byte regardless if this is a one or two byte
                 ; operand, because we'll need the LSB one way or the other.
                 ; We have a copy of the opcode on the stack, so we can now move
                 ; to the next byte
-                inc 4,x
-                bne +
-                inc 5,x                 ; ( addr+1 u 0 )
-+
-                lda 2,x
-                bne +
-                dec 3,x
-+
-                dec 2,x                 ; ( addr+1 u-1 0 )
+
+                jsr w_one
+                jsr w_slash_string
```

Comment:
this `/string` is a godsend which I didn't know existed

Reply:
Yes, that's one of those words that is very useful for processing strings, which is mildly painful in Forth, but not as bad as in some other languages.  I'm not sure what the best way to learn all the words is - I periodically review the glossary and sometimes learn about a word I didn't know about before (or need prior) that becomes a words I know/use often.